src/main/scala/clock/LinearEstimator.scala

```diff
@@ -2,11 +2,11 @@ package org.lichess.compression.clock
 
 object LinearEstimator:
 
-  @inline def encode(dest: Array[Int], startTime: Int): Unit =
```

Comment:
since scala3 we have an `inline` keyword that guarantees inlining https://docs.scala-lang.org/scala3/guides/macros/inline.html#inline-methods